docs/content/tutorials/eager_mode.md

```diff
@@ -0,0 +1,307 @@
+
+# Eager Mode (Aplpa) 
+
+Eager Mode is an interactive way to build superduper applications. 
+
+Users can input data as usual, continuously call models, and view results.
+Once the interactive debugging and construction are complete, 
+the corresponding data pipeline can be built directly through `apply`, 
+eliminating the need for direct debugging between AI application models and databases.
+
+:::note
+This feature is in the alpha version.
+:::
+
+Connect the database and insert data.
+
+```python
+from superduper import superduper
+
+db = superduper('mongomock://test')
+```
+
+<details>
+<summary>Outputs</summary>
+<pre>
+    2024-Jul-24 21:44:39.27| INFO     | zhouhaha-2.local| superduper.base.build:57   | Data Client is ready. mongomock.MongoClient('localhost', 27017)
+    2024-Jul-24 21:44:39.28| INFO     | zhouhaha-2.local| superduper.base.build:36   | Connecting to Metadata Client with engine:  mongomock.MongoClient('localhost', 27017)
+    2024-Jul-24 21:44:39.29| INFO     | zhouhaha-2.local| superduper.base.build:152  | Connecting to compute client: Compute(uri=None, compute_kwargs=\{\}, _path='superduper.backends.local.compute.LocalComputeBackend')
+    2024-Jul-24 21:44:39.29| INFO     | zhouhaha-2.local| superduper.base.datalayer:104  | Building Data Layer
+    2024-Jul-24 21:44:39.29| INFO     | zhouhaha-2.local| superduper.base.build:214  | Configuration: 
+     +---------------+------------------+
+    | Configuration |      Value       |
+    +---------------+------------------+
+    |  Data Backend | mongomock://test |
+    +---------------+------------------+
+
+</pre>
+</details>
+
+```python
+import numpy as np
+data = [
+    {"x": 1, "y": "2", "z": np.array([1, 2, 3])},
+    {"x": 2, "y": "3", "z": np.array([4, 5, 6])},
+    {"x": 3, "y": "4", "z": np.array([7, 8, 9])},
+]
+
+db["documents"].insert(data).execute()
+```
+
+<details>
+<summary>Outputs</summary>
+<pre>
+    2024-Jul-24 21:44:39.33| INFO     | zhouhaha-2.local| superduper.backends.mongodb.data_backend:226  | Table documents does not exist, auto creating...
+
+</pre>
+<pre>
+    ([ObjectId('66a105477a51db6bb84c3c5b'),
+      ObjectId('66a105477a51db6bb84c3c5c'),
+      ObjectId('66a105477a51db6bb84c3c5d')],
+     None)
+</pre>
+</details>
+
+When using `select.execute(eager_mode=True)`, all returned data will enter eager mode, which can be used for interactive model pipeline construction.
+
+```python
+datas = list(db["documents"].select().execute(eager_mode=True))
```

Comment:
Interesting language thing: plural of "data" is "data", singular is "datum".

Reply:
Hmm, Got it, I changed all of them